src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/GuidList.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.VisualStudio.OnlineEnvironment.Client
+{
+    internal class GuidList
```

Comment:
[CA1052](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1052?view=vs-2019) violation.

Reply:
Made the class static.